src/app/products/product-list/product-list.component.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {ProductListComponent} from "./product-list.component";
```

Comment:
There also should be tests for a service. I skipped at because of time. I had some issue with my GitHub account, because of I did not use long time ago.

Reply:
Also I have not seen a reason do add TestBed.